cpp/log/cert.cc

```diff
@@ -550,17 +550,17 @@ Cert::Status Cert::PublicKeySha256Digest(string* result) const {
     // is bad until proven otherwise.
     LOG(WARNING) << "Failed to compute public key digest";
     LOG_OPENSSL_ERRORS(WARNING);
-    return FALSE;
+    return util::Status(Code::INVALID_ARGUMENT, "SHA256 digest failed");
   }
   result->assign(reinterpret_cast<char*>(digest), len);
-  return TRUE;
+  return util::Status::OK;
 }
 
 
-Cert::Status Cert::SPKISha256Digest(string* result) const {
+util::Status Cert::SPKISha256Digest(string* result) const {
```

Comment:
I considered it. I think we should refactor all the similar patterns separately after Cert::Status has gone as the changes are already quite large. I'm open to changing my mind.